### PR TITLE
User endpoint

### PR DESCRIPTION
**Pull request by govi218**
## Notes
- added user endpoint, set up prettier format

## Tests
- check if user (and importantly `user.email`) are retrieved

### CODE REVIEW

File: core/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@crate.ai/discogs-sdk",
-  "version": "2.1.5",
+  "version": "2.2.3",
```
Comment (paprikaf) [inline core/package.json:3]: Not that I care much but how did you go from 2.1.5 to 2.2.3

File: core/tests/user.test.ts
Comment (paprikaf) [inline core/tests/user.test.ts:1]: I presume you ran pmpm test on this and all is good?

**Review by paprikaf — COMMENTED** [review]
overall LGTM. 
optional but I don't see an example of the user's endpoint in the  examples folder

File: core/package.json (hunk 1 above)
Comment (govi218) [reply core/package.json:3]: had some git shenanigans

**Pull request merged**